app/models/ticket_purchase.rb

```diff
@@ -58,4 +60,11 @@ def self.update_quantity(conference, quantity, ticket, user)
     purchase.quantity = quantity if quantity > 0
     purchase
   end
+
+  def pay(payment)
+    update_attributes(paid: true, payment_id: payment)
```

Comment:
Please check it again. You should be able to use `payment` directly as @mdeniz says.